src/main/java/org/apache/pirk/querier/wideskies/encrypt/EncryptQuery.java

```diff
@@ -163,8 +144,8 @@ public void encrypt(int numThreads) throws InterruptedException, PIRException
       query.generateExpTable();
     }
 
-    // Set the Querier object
-    querier = new Querier(selectors, paillier, query, embedSelectorMap);
+    // Answer the Querier object.
```

Comment:
Sure, I do tend to get creative when it is impl comments, as opposed to the javadoc :-)